src/docs/Welcome.mdx

```diff
@@ -7,7 +7,75 @@ import Icon from "../components/Icons/Icon";
 import Link from "../components/Link/Link";
 import Logo from "../components/Logo/Logo";
 import SimpleGrid from "../components/Grid/SimpleGrid";
+import Table from "../components/Table/Table";
 import Text from "../components/Text/Text";
+export const versionsHeadersData = [
+  "Version",
+  "Date",
+  "Summary",
+  "Documentation",
+  "Code",
+];
+export const versionsData = [
+  [
+    <Text size="body2" whiteSpace="nowrap">
+      Reservoir 2.1.6
+    </Text>,
+    <Text size="body2" whiteSpace="nowrap">
+      September 2023
```

Comment:
2.1.6 went out three weeks ago on Feb 22 (20 days ago from 3/13 based on npm).

Reply:
When I put this table together for the v3 branch, I used the dates that the major versions were first released (i.e. `1.0.0` and `2.0.0`).  So that begs the quetions, what is more important...the date of the initial major version release or the date of the final release within a major version?

Reply:
I would make it the date of that specific version. Should we poll it?

Reply:
No, I will use the exact dates for those versions.